sklearn/ensemble/tests/test_voting.py

```diff
@@ -513,6 +514,57 @@ def test_check_estimators_voting_estimator(estimator):
     check_no_attributes_set_in_init(estimator.__class__.__name__, estimator)
 
 
+def test_voting_verbose_vc(capsys):
```

Comment:
Since the 2 tests are really similar, I would advise making them general enough such that we can parametrize the test with a `VotingClassifier` and a `VotingRegressor`.